sdk/monitor/monitor-ingestion/review/monitor-ingestion.api.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@azure/monitor-ingestion"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { CommonClientOptions } from '@azure/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public (undocumented)
+export class FailedLogsIngestionError extends Error {
```

Comment:
Actually I am not using this anymore, we aren't throwing on complete failure anymore since we switched the design of the upload result. Umm, but should I "throw" on complete failure scenario with the "Upload Result" or just return the "Upload Result"?

Reply:
Removed this